### ISSUE TITLE
Stronger model of Katago change strength of calibrated rank?

### ISSUE THREAD

**Issue by exxelxior** [sanderland/katrain#700]
I don't know if this is the place for my question, if so, please excuse me.

I usually play against AI Calibrated Rank using the recommended model kata1-b18c384nbt-s6582191360-d3422816034 (2023-06-13)
I would like to know if I change the model for any of the latest **"b18"**, will the strength of a given Calibrated Rank be the same or stronger?

**Comment by HackYardo** [comment]
Yes, it is. The Calibrated Rank is designed for the best b15 (or b20) neutral network models at that time. The initial version of KaTrain is based on b15.
